kernel/arch/dreamcast/include/dc/video.h

```diff
@@ -47,9 +47,13 @@ __BEGIN_DECLS
 */
 #define PM_RGB555   0       /**< \brief RGB555 pixel mode (15-bit) */
 #define PM_RGB565   1       /**< \brief RGB565 pixel mode (16-bit) */
+#define PM_UNK      2       /**< \brief Unknown pixel mode (?-bit) */
```

Comment:
Look here:

https://github.com/DC-SWAT/DreamShell/blob/master/firmware/isoldr/loader/kos/dc/video.h#L49

Reply:
I use it for screenshot feature in ISO Loader.